src/translations/notepadqq_pl.ts

```diff
@@ -1,155 +1,423 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
 <TS version="2.1" language="pl_PL">
+<context>
+    <name>AdvancedSearchDock</name>
+    <message>
+        <location filename="../ui/Search/advancedsearchdock.cpp" line="161"/>
+        <location filename="../ui/Search/advancedsearchdock.cpp" line="651"/>
+        <source>Advanced Search</source>
+        <translation>Wyszukiwanie zaawansowane</translation>
+    </message>
+    <message>
+        <location filename="../ui/Search/advancedsearchdock.cpp" line="166"/>
+        <source>Clear Search History</source>
+        <translation>Wyczyść historię wyszukiwania</translation>
+    </message>
+    <message>
+        <location filename="../ui/Search/advancedsearchdock.cpp" line="175"/>
+        <source>Go To Previous Result (Shift+F4)</source>
+        <translation>Przejdź do poprzedniego wyniku (Shift+F4)</translation>
+    </message>
+    <message>
+        <location filename="../ui/Search/advancedsearchdock.cpp" line="180"/>
+        <source>Go To Next Result (F4)</source>
+        <translation>Przejdź do następnego wyniku (Shift+F4)</translation>
```

Comment:
I think this is supposed to be `(F4)`, since the previous one is already `(Shift+F4)`